### PR TITLE
Changes in SUMA integration

### PR DESCRIPTION
**Pull request by abravosuse**
This PR must be reviewed but cannot be merged/published until the new version of Trento is out. It describes the second iteration of the integration of Trento with SUSE Manager, by which we can see the list of available patches and  upgradable packages for each registered host that is also managed by SUMA.

### PR creator: Description

Describe the overall goals of this pull request.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### CODE REVIEW

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -1784,9 +1783,46 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+        </mediaobject>
+      </figure>
+      <para>Clicking on the Upgradable Packages box takes us to the Upgradable Package overview,
+    where the user can see the list of packages that can be upgraded in that particular host:</para>
+           <figure>
+         <title><guimenu>Upgradable Packages overview</guimenu></title>
+        <mediaobject>
+          <imageobject>
+            <imagedata
+              fileref="trento-upgradable-packages-overview.png" />
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1804]: This image is missing according to the GitHub Action.

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -1770,11 +1769,11 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
         </mediaobject>
       </figure>  
    <para>
-   With the &suma; settings configured, the &sap; admin can test the connection
-   by clicking <guimenu>Test Connection</guimenu>.
-   If the connection is successful, the Host Details view displays a summary
-   of the available patches and the upgradable packages for each host
-   managed by &suma;:</para>
+   Onc the &suma; settings are configured, the &sap; administrator can test the connection
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1772]: Typo:

```suggestion
   Once the &suma; settings are configured, the &sap; administrator can test the connection
```

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -1770,11 +1769,11 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+   by clicking the corresponding button.
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1773]: What do you mean by "the corresponding button"? Is the button labeled "Test"? In that case, use:

```suggestion
   by clicking the <guimenu>Test</guimenu> button.
```

If you don't like that repeats "test" twice, you can use replace the verb "test" with "check".

File: trento/xml/article_sap_trento.xml [hunk 4]
```diff
@@ -1770,11 +1769,11 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+   If the connection is successful, the Host Details view of each host
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1774]:
```suggestion
   If the connection is successful, the <guimenu>Host Details</guimenu> view of each host
```

File: trento/xml/article_sap_trento.xml [hunk 5]
```diff
@@ -1784,9 +1783,46 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
           </imageobject>
         </mediaobject>
       </figure>
-   <para>There are three types of available patches: security advisories, bug fixes and feature
-   enhancement. Security advisories are regarded as critical. If one is found,
-   the health of the host will be set to critical too. The others are to be seen as warnings.</para>
+   <para>Clicking on the Relevant Patches box takes us to the Relevant Patches overview,
+    where the user can see the list of patches available for that particular host:</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1787]:
```suggestion
   <para>Clicking on the <guimenu>Relevant Patches</guimenu> box takes us to the <guimenu>Relevant Patches</guimenu> overview,
    where the user can see the list of patches available for that particular host:</para>
```

File: trento/xml/article_sap_trento.xml [hunk 6]
```diff
@@ -1784,9 +1783,46 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+         <title><guimenu>Available Patches overview</guimenu></title>
+        <mediaobject>
+          <imageobject>
+            <imagedata
+              fileref="trento-available-patches-overview.png" />
+          </imageobject>
+        </mediaobject>
+      </figure>
+      <para>Clicking on the Upgradable Packages box takes us to the Upgradable Package overview,
+    where the user can see the list of packages that can be upgraded in that particular host:</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1798]:
```suggestion
      <para>Clicking on the <guimenu>Upgradable Packages</guimenu> box takes us to the <guimenu>Upgradable Package</guimenu> overview,
    where the user can see the list of packages that can be upgraded in that particular host:</para>
```

File: trento/xml/article_sap_trento.xml [hunk 7]
```diff
@@ -1784,9 +1783,46 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+        <mediaobject>
+          <imageobject>
+            <imagedata
+              fileref="trento-upgradable-packages-overview.png" />
+          </imageobject>
+        </mediaobject>
+      </figure>
+   <para>By clicking on an advisory or patch link, the user will access the corresponding details view,
+    with relevant information about it such us whether it requires a reboot or not, associated 
+    vulnerabilities, or a list of affected hosts:</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1810]: Small rephrase and typo:

```suggestion
   <para>By clicking on an advisory or patch link, the user will access the corresponding details view. This view shows
   relevant information such as whether it requires a reboot or not, associated 
    vulnerabilities, or a list of affected hosts:</para>
```

File: trento/xml/article_sap_trento.xml [hunk 8]
```diff
@@ -1784,9 +1783,46 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+         <title><guimenu>Advisory Details view</guimenu></title>
+        <mediaobject>
+          <imageobject>
+            <imagedata
+              fileref="trento-advisory-details-view.png" />
+          </imageobject>
+        </mediaobject>
+      </figure>  
+   <para>There are three types of patches or advisories: security advisories, bug fixes and feature
+   enhancements. Security advisories are regarded as critical. If one is avaiable,
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1821]: Missing Oxford comma. :wink: 

```suggestion
   <para>There are three types of patches or advisories: security advisories, bug fixes, and feature
   enhancements. Security advisories are regarded as critical. If one is avaiable,
```

File: trento/xml/article_sap_trento.xml [hunk 9]
```diff
@@ -1784,9 +1783,46 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+   the health of the host will be set to critical. IF there are availale patches but none of them
+    is a security one, the health of the host will switch to warning. When a host cannot be found in
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1823]: Typos:

```suggestion
   the health of the host will be set to critical. If there are available patches but none of them
    is a security one, the health of the host will switch to warning. When a host cannot be found in
```

**Review by tomschr — CHANGES_REQUESTED** [review]
@abravosuse Some small issues and a missing image. :slightly_smiling_face:

**Review by abravosuse — COMMENTED** [review]
@tomschr missing file uploaded. All other suggestions accepted AS-IS. Thank you!

**Review by abravosuse — COMMENTED** [review]
LGTM!

File: trento/xml/article_sap_trento.xml [hunk 10]
```diff
@@ -1770,11 +1769,11 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
-   With the &suma; settings configured, the &sap; admin can test the connection
-   by clicking <guimenu>Test Connection</guimenu>.
-   If the connection is successful, the Host Details view displays a summary
-   of the available patches and the upgradable packages for each host
-   managed by &suma;:</para>
+   Once the &suma; settings are configured, the &sap; administrator can test the connection
+   by clicking the <guimenu>Test</guimenu> button.
+   If the connection is successful, the <guimenu>Host Details</guimenu> view of each host
+   managed by &suma; will display a summary
+   of available patches and upgradable packages:</para>
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:1776]:
```suggestion
   If the connection is successful, the <guimenu>Host Details</guimenu> view of each host
   managed by &suma; displays a summary
   of available patches and upgradable packages:</para>
```

File: trento/xml/article_sap_trento.xml [hunk 11]
```diff
@@ -1784,9 +1783,46 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
           </imageobject>
         </mediaobject>
       </figure>
-   <para>There are three types of available patches: security advisories, bug fixes and feature
-   enhancement. Security advisories are regarded as critical. If one is found,
-   the health of the host will be set to critical too. The others are to be seen as warnings.</para>
+   <para>Clicking on the <guimenu>Relevant Patches</guimenu> box takes us to the <guimenu>Relevant Patches</guimenu> overview,
+    where the user can see the list of patches available for that particular host:</para>
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:1787]:
```suggestion
   <para>Click on <guimenu>Relevant Patches</guimenu> to open the <guimenu>Relevant Patches</guimenu> overview containing a list of patches available for that particular host:</para>
```

File: trento/xml/article_sap_trento.xml [hunk 12]
```diff
@@ -1784,9 +1783,46 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+         <title><guimenu>Available Patches overview</guimenu></title>
+        <mediaobject>
+          <imageobject>
+            <imagedata
+              fileref="trento-available-patches-overview.png" />
+          </imageobject>
+        </mediaobject>
+      </figure>
+      <para>Clicking on the <guimenu>Upgradable Packages</guimenu> box takes us to the <guimenu>Upgradable Package</guimenu> overview,
+    where the user can see the list of packages that can be upgraded in that particular host:</para>
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:1798]:
```suggestion
      <para>Click <guimenu>Upgradable Packages</guimenu> to open the <guimenu>Upgradable Package</guimenu> overview containing a list of packages that can be upgraded on that particular host:</para>
```

File: trento/xml/article_sap_trento.xml [hunk 13]
```diff
@@ -1784,9 +1783,46 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+        <mediaobject>
+          <imageobject>
+            <imagedata
+              fileref="trento-upgradable-packages-overview.png" />
+          </imageobject>
+        </mediaobject>
+      </figure>
+   <para>By clicking on an advisory or patch link, the user will access the corresponding details view. This view shows
+   relevant information such as whether it requires a reboot or not, associated 
+    vulnerabilities, or a list of affected hosts:</para>
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:1810]:
```suggestion
   <para> Click  on an advisory or patch link to access the corresponding details view. This view shows
   relevant information, such as whether it requires a reboot, associated 
    vulnerabilities, or a list of affected hosts:</para>
```

File: trento/xml/article_sap_trento.xml [hunk 14]
```diff
@@ -1784,9 +1783,46 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+            <imagedata
+              fileref="trento-advisory-details-view.png" />
+          </imageobject>
+        </mediaobject>
+      </figure>  
+   <para>There are three types of patches or advisories: security advisories, bug fixes, and feature
+   enhancements. Security advisories are regarded as critical. If one is avaiable,
+   the health of the host will be set to critical. If there are available patches but none of them
+    is a security one, the health of the host will switch to warning. When a host cannot be found in
+    &suma; or there is a problem retrieving the data for it, its health is set as unknnown.
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:1824]:
```suggestion
   <para>There are three types of patches or advisories: security advisories, bug fixes, and feature
   enhancements. Security advisories are considered as critical. The health status of the host is set to critical if there is a security advisory. If there are patches available, but none of them
    is a security-related, the health status of the host switches to warning. When a host cannot be found in
    &suma;, or there is a problem retrieving the data for it, its health status is set as unknnown. 
```

File: trento/xml/article_sap_trento.xml (hunk 12 above)
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:1797]:
```suggestion
      <para>Click <guimenu>Upgradable Packages</guimenu> to open the <guimenu>Upgradable Packages</guimenu> overview containing a list of packages that can be upgraded on that particular host:</para>>
```

**Review by abravosuse — COMMENTED** [review]
Thanks for the suggestions @dmpop . Just a small change request from my side: Upgradable Package > Upgradable Packages.

**Review by tomschr — APPROVED** [review]

**Review by tomschr — APPROVED** [review]
Sounds good! :+1:

**Pull request merged**